lib/_http_outgoing.js

```diff
@@ -283,17 +285,26 @@ OutgoingMessage.prototype._send = function _send(data, encoding, callback) {
 OutgoingMessage.prototype._writeRaw = _writeRaw;
 function _writeRaw(data, encoding, callback) {
   const conn = this.socket;
-  if (conn && conn.destroyed) {
-    // The socket was destroyed. If we're still trying to write to it,
-    // then we haven't gotten the 'close' event yet.
-    return false;
-  }
 
   if (typeof encoding === 'function') {
     callback = encoding;
     encoding = null;
   }
 
+  if (conn && conn.destroyed) {
+    const err = new ERR_STREAM_DESTROYED('write');
+    if (callback) {
+      callback(err);
+    }
+    if (this.listenerCount('error') > 0 && this[kErrored] === undefined) {
+      this[kErrored] = true;
```

Comment:
Is there a specific reason to silence all errors besides the first one?

Reply:
@BridgeAR: that's how streams do it, see `destroy.js`.

Reply:
The reason is that this will normally be called 4 times in case this condition is met, as a response is composed of 4 chunks.